client/cl_functions.lua

```diff
@@ -90,14 +93,6 @@ function notifyRetrievePrompt(garage, id)
                 queue = "id"
             })
         end
-        if Config.esxNotify then
```

Comment:
This needs to be here otherwise there is no way to tell the player they should retrieve the vehicle.